--- PKGBUILD	2014-03-13 18:19:49.000000000 -0500
+++ PKGBUILD.ldap	2014-03-20 11:06:28.175957724 -0500
@@ -23,7 +23,7 @@
 license=('custom')
 
 # Packages that your software needs to run. If the dependancy requires a minimum version number use the >= operator
-depends=('libevent>=2.0' 'libzdb>=2.10.2' 'libsearpc>=1.2.1' 'jansson>=2.2.1')
+depends=('libevent>=2.0' 'libzdb>=2.10.2' 'libsearpc>=1.2.1' 'jansson>=2.2.1' 'libldap')
 
 # Packages that must be installed to build the software, but at not necessary to run it
 makedepends=('vala>=0.8' 'pacman>=4.1')
@@ -48,7 +48,7 @@
 
 configure_ccnet() {
 	./autogen.sh
-	CFLAGS="-lpthread" ./configure --enable-server --prefix=/usr PYTHON=/usr/bin/python2
+	CFLAGS="-lpthread" ./configure --enable-server --enable-ldap --prefix=/usr PYTHON=/usr/bin/python2
 }
 
 pkgver() {
